contractcourt/breach_arbitrator.go

```diff
@@ -1466,6 +1526,15 @@ func (b *BreachArbitrator) createSweepTx(inputs ...input.Input) (*wire.MsgTx,
 	// nLockTime, and output are already included in the TxWeightEstimator.
 	weightEstimate.AddP2TROutput()
 
+	// If any of our inputs has a resolution blob, then we'll add another
+	// P2TR output.
+	hasBlobs := fn.Any(func(i input.Input) bool {
+		return i.ResolutionBlob().IsSome()
+	}, inputs)
+	if hasBlobs {
```

Comment:
The goal is to extract the assets that are swept into their own output (separate assets from BTC). So there's only _one_ additional output necessary. Updated the comment to be more clear.

Reply:
If we have to split out all the HTLC outputs individually due to our sweep-all justice txn not confirming in time, that will mean an equal number of extra outputs. Just bringing it up so that you're aware.

Reply:
Yeah that will be covered in an upcoming portion @Crypt-iQ. As is, this only works with the default sweep scenario where we just need to sweep the commitment output.